src/controllers/CustomersController.php

```diff
@@ -204,16 +204,16 @@ public function actionCustomersTable(): Response
 
             $rows[] = [
                 'id' => $customer['id'],
-                'title' => $customer['email'],
+                'title' => htmlspecialchars($customer['email']),
```

Comment:
Everywhere you're using `htmlspecialchars`, it should be swapped to `craft\helpers\Html::encode()`

Reply:
Fixed with 9a8c556c898dcdce1d3fa24f0a5814b31d33d3fa